Library/Formula/fltk.rb

```diff
@@ -14,20 +32,15 @@ class Fltk < Formula
   depends_on "libpng"
   depends_on "jpeg"
 
-  # Fltk 1.3.4 include support for El Capitan. Remove on update.
-  depends_on MaximumMacOSRequirement => :yosemite
-
   fails_with :clang do
     build 318
     cause "http://llvm.org/bugs/show_bug.cgi?id=10338"
   end
 
-  # Fixes issue with -lpng not found.
-  # Based on: https://trac.macports.org/browser/trunk/dports/aqua/fltk/files/patch-src-Makefile.diff
-  patch :DATA
-
   def install
     ENV.universal_binary if build.universal?
+
+    system "NOCONFIGURE=1 ./autogen.sh" if build.devel?
```

Comment:
`system "foo bar"` runs in a shell while `system "foo", "bar"` doesn’t. We thus prefer the later.